### PR TITLE
Deprecation

### PR DESCRIPTION
**Pull request by camilamacedo86**
# Jira link(s)
https://issues.jboss.org/browse/RHMAP-21138

# What
Add deprecation notice

# Why
Analytics feature is deprecated

# How
Update the README to make it clear. 

# Verification Steps

## Checklist:

- [ ] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 #fh-messaging(1) -- The FeedHenry Message Server
```
Comment (laurafitzgerald) [inline README.md]: @camilamacedo86 this change looks fine. Just a small note that the first header is not rendering as there is a space missing after the #. Could that be updated as part of this pr also?

Comment (camilamacedo86) [reply README.md]: Done:-)

**Review by davidffrench — APPROVED** [review]
LGTM. 

We will not be able to archive the repo itself due to it still being included in releases.

**Comment by camilamacedo86** [comment]
@davidffrench thank you for your review.

**Pull request merged**